less/core/drawer.less

```diff
@@ -22,7 +22,7 @@
 }
 
 .drawer:not(.disable-animation) {
-  .transition(left @duration ease-in, right @duration ease-in, transform @duration ease-in, opacity @duration ease-in;);
+  .transition(left var(--drawerDuration) @animation-easing, right var(--drawerDuration) @animation-easing, transform var(--drawerDuration) @animation-easing, opacity var(--drawerDuration) @animation-easing;);
```

Comment:
I don't think any of our other variables are camel case or without the `--adapt-` prefix `--adapt-drawer-animation-duration`. 

What is the `@duration` property for if not this? Why do we need a differentiating variable independent of the drawer animation timing?

https://github.com/search?q=repo%3Aadaptlearning%2Fadapt-contrib-vanilla+%40duration&type=code&p=1

Reply:
Renamed the drawer variable to `--adapt-drawer-duration`.

> What is the @duration property for if not this? Why do we need a differentiating variable for notify animation duration independent of the drawer animation duration?

I commented on this [here](https://github.com/adaptlearning/adapt-contrib-core/pull/601#discussion_r1823435933).